eGovPDX/portlandor
# POWR-2123: Fix News Morsel Pattern

**This PR fixes two issues:**

1) An issue where the Drupal Twig `render` filter was breaking the Pattern Lab Twig compilation process. Here I've simply moved the `render` into the environment where it's actually present (Drupal theme engine).
2) Clears up an annoying console error whenever compiling a watch command (couldn't compile Pattern Lab Morsel demos)

Let me know if you'd like to discuss at all! :-)

**To Test**
1. **Navigate to —>** https://powr-2123-2-portlandor.pantheonsite.io/pattern-lab/?p=viewall-components-morsel
2. **Verify —>** The morsel pattern loads up just fine!
3. **To test the updated functionality —>** You'll need to edit a news item and provide the correct updated date/time, save, then view the list
